tests/test-performance.js

```diff
@@ -0,0 +1,26 @@
+var performance = require("performance");
+
+var total = 0;
+var passed = 0;
+
+function assert(actual, description) {
+    total += 1;
+    var label = "\033[1m\033[31mFAIL\033[0m";
+    if (actual === true) {
+        passed += 1;
+        label = "\033[1m\033[32mPASS\033[0m";
+    }
+    console.log(label + " - " + description);
+}
+
+var before = performance.now();
+//console.log("performance.now() before:", before);
```

Comment:
I will. But note that the moment the test fails, someone would need to re-add this code to diagnose it. Or just uncomment it, which saves time. Apparently, that already helped @grgustaf in https://github.com/01org/zephyr.js/pull/346#discussion_r86405975 . Of course, this version should be final and never fail. But, I thought the same about my original version too, and @grgustaf had problems.